src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -429,3 +434,21 @@
           text: >
             az acr build -r MyRegistry .
 """
+
+helps['acr image import'] = """
+    type: command
+    short-summary: Imports an image to the container registry from the specified container registry.
```

Comment:
Is this basically copying from one registry to another or is it from any source to an ACR?  It sounds like "copy" might be the more appropriate verb than "import".  Import usually implies converting a file into resource, so there's a transformation. This sounds like just copying an image into the registry.  But I could be misinterpreting.